packages/mesos-metrics-module/build

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+mkdir -p build
+pushd build
+
+# configure module build with cmake
+cmake \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DTESTS_ENABLED=False \
+  -DCMAKE_INSTALL_PREFIX=$PKG_PATH \
+  -Dmesos_INCLUDE_DIR=/opt/mesosphere/include \
+  -Ddefault_BIN_DIR=/opt/mesosphere/bin \
+  -Ddefault_LIB_DIR=/opt/mesosphere/lib \
+  -DUSE_LOCAL_PROTOBUF=False \
+  -Dprotobuf_INCLUDE_DIR=/opt/mesosphere/lib/mesos/3rdparty/include \
```

Comment:
Since this is a mesos module, why can't we just use a configuration flag which says "here's all the mesos bundled includes" rather than having to call them out one by one?